libs/net/dal/Services/ContentService.cs

```diff
@@ -115,12 +124,40 @@ public IPaged<Content> Find(ContentFilter filter)
         query = query.Skip(skip).Take(filter.Quantity);
 
         var items = query?.ToArray() ?? Array.Empty<Content>();
+
+        HandleInvalidEncoding(items);
+
         return new Paged<Content>(items, filter.Page, filter.Quantity, total);
     }
 
+    private void HandleInvalidEncoding(params Content[] contents)
```

Comment:
Running this method every time we request content on every single content items is not a good design.  It will result in additional memory and CPU every single request.  Why would we ever want to do this?

Reply:
This has been updated, thanks